### ISSUE TITLE
Getting stuck on 'copying' with file paths without privileges

### ISSUE THREAD

**Issue by OfficialKris** [overmindstudios/BlenderUpdater#28]
Hi, I'm getting stuck on 'copying' with the path name set to blender inside C:\Program Files\Blender Foundation\Blender on windows 10.

This is only fixed by running the program in admin mode.

A warning would be nice, or a button to run the program in admin mode and close the non-admin mode instance, or both.

This also relates to another bug. The program says the selected version is already installed when you run the program again, even though it isn't.

**Comment by tobkum** [comment]
Ah, yes... that is a known issue as of now, also stated in the Readme. But I'll look into the idea of checking if admin priviledges are needed and prompting the user if need be.
Thanks!
-T

**Comment by zutxita** [comment]
Yeah, it happens. Also, when you get stucked in Copying... and you close and run it again (as admin), but you already have done the dowloading and extraction succesfully... it could be nice that it run directly to the copying part, to speed up a little.
